conf/application.conf

```diff
@@ -140,8 +140,8 @@ microservice {
     eis {
           protocol = "http"
           host = "localhost"
-          uri = "/common-transit-convention-trader-at-destination/message-notification"
-          port = 9481
+          uri = "/common-transit-convention-trader-at-departure/movements/departures"
```

Comment:
This should probably be "/transits-movements-trader-at-departure-stub/movements/departures"